core/unix/module_elf.c

```diff
@@ -1085,6 +1085,9 @@ module_lookup_symbol(ELF_SYM_TYPE *sym, os_privmod_data_t *pd)
      */
     ASSERT_OWN_RECURSIVE_LOCK(true, &privload_lock);
     mod = privload_first_module();
+    /* Symbols are currently looked up following the dependency chain depth-first instead
```

Comment:
This should be an XXX (or FIXME).    Please mention that the ELF spec requires breadth-first symbol lookup.

Reply:
Added "FIXME i#3850".